Program.cs

```diff
@@ -18,42 +17,132 @@ public int add(string Numbers)
                 return 0;
             }
 
         }
-        private int tryAdd(string Numbers)
+        private int tryAddSimple(string Numbers)
         {
             int ans;
             ans = Numbers.Where(x => x != ',').Aggregate((sum, i) => sum += i);
             ans -= Numbers.Where(x => x != ',').Count() * '0';
             return ans;
         }
+        private int tryAddbig(string Numbers)
+        {
+            List<string> Nums = Numbers.Split(',').ToList();
+            int sum = GetSum(Nums);
+            return sum;
+        }
         private int tryAddWithEndLine(string Numbers)
         {
-            int ans;
-            string endLine = "/n";
-            Numbers = Numbers.Replace("/n", "");
-            ans = Numbers.Where(x => x != ',').Aggregate((sum, i) => sum += i);
-            ans -= Numbers.Where(x => x != ',').Count() * '0';
-            return ans;
+            Numbers = Numbers.Replace("/n", ",");
+            List<string> Nums = Numbers.Split(',').ToList();
+            int sum = GetSum(Nums);
+            return sum;
         }
-         private int tryAdddelimiters(string Numbers)
+        private int tryAdddelimiters(string Numbers)
         {
-            int ans;
-            char endLine = Numbers[2];
-            Numbers = Numbers.Replace("/n", "");
-            Numbers = Numbers.Replace("//", "");
-            ans = Numbers.Where(x => x != endLine).Aggregate((sum, i) => sum += i);
-            ans -= Numbers.Where(x => x != endLine).Count() * '0';
-            return ans;
+            char diameter = ',';
+            if (Numbers.StartsWith("//"))
+            {
+                diameter = Getdiameter(Numbers);
+            }
+            List<string> Nums = Numbers.Split(diameter).ToList();
+            int sum = GetSum(Nums);
+            return sum;
+        }
+        private int tryAdddelimitersWithNegitave(string Numbers)
+        {
+            char diameter = ',';
+            if (Numbers.StartsWith("//"))
+            {
+                diameter = Getdiameter(Numbers);
+            }
+            RemoveNewLine(Numbers, diameter);
+            List<string> Nums = Numbers.Split(diameter).ToList();
+            testNegitaves(Nums);
+            int sum = GetSum(Nums);
+            return sum;
+        }
+        private int tryAdddelimitersWithNegitaveIgnoreingBigNumbers(string Numbers)
+        {
+            char diameter = ',';
+            if (Numbers.StartsWith("//"))
+            {
+                diameter = Getdiameter(Numbers);
+            }
+            RemoveNewLine(Numbers, diameter);
+            List<string> Nums = Numbers.Split(diameter).ToList();
+            testNegitaves(Nums);
+            FilterNums(Nums);
+            int sum = GetSum(Nums);
+            return sum;
+        }
+
+        private void FilterNums(List<string> nums)
+        {
+            for (int i = 0; i < nums.Count(); i++)
+            {
+                if (nums.ElementAt(i).Count() > 3)
```

Comment:
this is a hacky why 
consider the users says bigger than 4500 then you need to change the whole method
and the filter method doesn't clearly show what is doing 
you need to run the code or read through it carefully to figure out the condition that numbers should be less than 1000

Reply:
also why the use of ElementAt 
couldn't we just use [] operator ?